chart/templates/service_account.yaml

```diff
@@ -6,10 +6,10 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: chaos-controller
-  namespace: chaos-engineering
+  namespace: {{ .Values.global.namespace }}
```

Comment:
Is this still going to default to chaos-engineering for us?

Reply:
oops, thanks for spotting. Updated.
The challenge of keeping internal charts in sync with the Open Source. With these changes we'll be fully in sync :)